pkg/network/fuzz_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build go1.18
+//go:build go1.19
```

Comment:
These can be dropped completely. They were only relevant pre-1.18.

Reply:
Fixed.